spec/presenters/user_presenter_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "rails_helper"
+
+RSpec.describe UserPresenter do
+  subject { described_class.new(user) }
+  let(:global_role_list) { subject.global_role_list }
+  let(:global_role_select_options) { subject.global_role_select_options }
+
+  shared_examples_for "it has no global roles" do
+    it { expect(global_role_list).to eq("") }
+    it { expect(global_role_select_options).not_to include('selected="selected"') }
+  end
+
+  shared_examples_for "it has one global role" do
+    it { expect(global_role_list).to eq("Administrator") }
+    it { expect(global_role_select_options).to include('selected="selected">Administrator') }
+    it { expect(global_role_select_options).not_to include('selected="selected">Billing Administrator') }
+  end
+
+  shared_examples_for "it has multiple global roles" do
+    it { expect(global_role_list).to eq("Administrator, Billing Administrator") }
+    it { expect(global_role_select_options).to include('selected="selected">Administrator') }
+    it { expect(global_role_select_options).to include('selected="selected">Billing Administrator') }
+  end
+
+  context "when the user has no global roles" do
+    let(:user) { create(:user) }
+
+    it_behaves_like "it has no global roles"
```

Comment:
When we're only using each of the shared_examples once, it almost becomes a little harder to understand the spec because you have to looks in multiple places, and the wording is kind of weird "when the user has no global roles it has no global roles". sort of becomes a tautology. I'd be inclined towards putting the specs directly within the individual contexts.